dss/index/es/schemainfo.py

```diff
@@ -0,0 +1,25 @@
+import re
+from collections import namedtuple
+from typing import Optional
+
+from dss.util.types import JSON
+
+schema_url_regex = re.compile(r'/(?P<major_version>[0-9]+)\.[0-9]+\.[0-9]+[\w\d/]*/(?P<type>[\w\d]+)(.json)?$')
```

Comment:
Can you add a comment with an example URL before the regex? It would help interpreting the URL.

Reply:
added